src/components/CloudHookForm.vue

```diff
@@ -165,6 +165,10 @@ export default {
       return allHooks.filter(
         t => (t.requiresCloud && this.isCloud) || !t.requiresCloud
       )
+    },
+    projectNamePrefix() {
```

Comment:
Could this be moved to `methods` instead? I'm not sure it needs to be a computed prop.

Reply:
Yeah, I'm still learning more about the difference between methods vs computed prop but I thought it might be better because it only runs again if a reactive dependency changes but in this case, I don't think it makes too much of a difference. I'll move it to methods.